backend/frontend/tests/test_restricted_user_board_access.py

```diff
@@ -0,0 +1,156 @@
+from guardian.shortcuts import assign_perm
+from core.models import Team
+from core.tests.factories import UserFactory, TeamFactory
+from .frontend_test_mixin import FrontendTestMixin
+
+
+class TestUserBoardControlledAccess(FrontendTestMixin):
+    def setUp(self):
+        super().setUp()
+
+        self.viewed_user_team = TeamFactory()
+        self.viewed_user = UserFactory(
+            email="learner.umuzi.org", is_staff=True  # TODO: remove this
```

Comment:
A forgotten todo? If it should be there please leave a reason or more info just like login todo on line 117.

Reply:
Not really, more of a reminder. After we merge the proper login urls we wont need is_staff=True. This is sort of a workwround for getting a valid session via the admin portal